package.json

```diff
@@ -76,6 +76,7 @@
         "@vitejs/plugin-react": "^3.1.0",
         "eslint": "^8.33.0",
         "eslint-config-prettier": "^8.6.0",
+        "eslint-import-resolver-typescript": "^3.5.3",
```

Comment:
This was to fix some errors I was getting as described by this SO post with the same problem: https://stackoverflow.com/questions/69446204/resolve-error-typescript-with-invalid-interface-loaded-as-resolver-eslint

Reply:
Oh my bad, I had the same issue but I forgot to commit that to the linting config update